Changelog.md

```diff
@@ -7,6 +7,7 @@
 ### ✨ New features and improvements
 - Added new loading spinner icon for tables (#7602)
 - Update message and page displaying cannot create new course via external LTI tool (#7669)
+- Prevent grade change when scrolling up or down with mouse or keys (#7680)
```

Comment:
Let's categorize this as a bug fix. Also, the current entry does not describe where in the UI this change effects. UI changes should always include context about what component/view/area is being affected.